pyp5js/http/web_app.py

```diff
@@ -14,6 +14,8 @@
 @app.route("/")
 def sketches_list_view():
     sketches = []
+    if not SKETCHBOOK_DIR.exists():
+        SKETCHBOOK_DIR.mkdir()
```

Comment:
The first error that I faced while running the tests was the directory was not created, so I had to copy this lines from the config.py file into the `sketches_list_view()`. Need to check if this is a good solution and if following this approach we want to remove those lines from the config.py file.

Reply:
A suggestion here is to create a `pytest.fixture` to always create this directory and avoid doing this check in the view and directory creation inside the view. This error you've had can be somehow related to an import order in the tests or stuff like that. Because when running pyp5js "for real" this directory will always be created given the import of the `pyp5js.config` module in the very first entry point of the lib.

My suggestion is for you to use the [Flask-Testing TestCase methods](https://github.com/berinhard/pyp5js/blob/develop/pyp5js/tests/test_commands.py#L13) `setUp` and `tearDown`  to create a base test class for pyp5js web app. Something like:

```python
class Pyp5jsWebTestCase(TestCase):

    def create_app(self):
        app = web_app
        app.config['TESTING'] = True
        return app    

    def setUp(self):
        if not SKETCHBOOK_DIR.exists():
            SKETCHBOOK_DIR.mkdir()

    def tearDown(self):
        if SKETCHBOOK_DIR.exists():
            shutil.rmtree(SKETCHBOOK_DIR)
```

Reply:
I think the problem with having this condition in the `setUp` and `tearDown` methods is that we aren't really testing the creation of the dir, we are assuming that it is granted and therefore the part of the code in which it exists (the `config.py` file) stays uncovered. So even if  someone erases those lines from the `config.py` file, all the tests will still be passing.

Reply:
But at the same time I still do not think that the responsibility of creating the dir should be given to `sketches_list_view`, so I am really hesitating on where it should be placed

Reply:
I think you're totally correct about your points and your doubt exposes a flaw in `pyp5js`'s architecture. The `config` should be a lazy-load object instead of a python module as now. To unblock you from that, I've created #94 to address a refactor on how pyp5js manages configuration.

So, let's go with the `setUp`/`tearDown` approach for now so we can have a better test coverage that'll help with the config refactoring. Is that ok for you?

Reply:
Sure!